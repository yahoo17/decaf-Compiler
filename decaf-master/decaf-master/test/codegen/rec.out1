20000
19000
18000
17000
Can't expand stack segment by 24 bytes to 524288 bytes
Use -lstack # with # > 524288
 
 
