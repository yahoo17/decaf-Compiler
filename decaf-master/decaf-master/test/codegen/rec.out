20000
19000
18000
17000
16000
Can't expand stack segment by 8 bytes to 524288 bytes
Use -lstack # with # > 524288
